lib/grpc/GrpcServer.ts

```diff
@@ -70,6 +85,56 @@ class GrpcServer {
       });
     });
   }
+
+  private generateCertificate = (tlsCertPath: string, tlsKeyPath: string) => {
```

Comment:
Maybe add a brief method comment block here explaining that this generates a certificate and saves it to disk, the latter part is not obvious from the name.